openvdb/openvdb/CMakeLists.txt

```diff
@@ -505,8 +505,7 @@ list(APPEND OPENVDB_CORE_PRIVATE_DEFINES -DOPENVDB_PRIVATE)
 if(WIN32)
   if(USE_IMATH_HALF)
     # @note OPENVDB_OPENEXR_STATICLIB is old functionality and should be removed
-    if(ILMBASE_USE_STATIC_LIBS OR (${ILMBASE_Half_LIB_TYPE} STREQUAL STATIC_LIBRARY) OR
-       IMATH_USE_STATIC_LIBS OR (${IMATH_LIB_TYPE} STREQUAL STATIC_LIBRARY))
```

Comment:
So this failed for me on Windows because `IMATH_LIB_TYPE` isn't a think. @cary-ilm I see that Imath checks for a `Imath_DLL` define. Is this automatically defined on Windows for us if Imath is built as a DLL? Or do downstream projects still need to define this themselves?